index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
Your project looks great! However, there is a small issue. 

- [ ] You need to personalize your README.md file as the given [template](https://github.com/microverseinc/readme-template). I would kindly suggest you use the Microverse [template](https://github.com/microverseinc/readme-template) and personalize it for your project. And one more thing.. Do not forget to add and personalize [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md) license to it.  😉